### ISSUE TITLE
Squiz.Functions.MultiLineFunctionDeclaration.Ident has incorrect indent warning

### ISSUE THREAD

**Issue by TomasVotruba** [squizlabs/PHP_CodeSniffer#843]
This code

```php
public function __construct(
    ControllerResolverInterface $controllerResolver
}
```

will return:

```
[x] Multi-line function declaration not indented correctly;
    |       |     expected 5 spaces but found 8
```

With tabs I recall it worded correctly. I use spaces now.

**Comment by TomasVotruba** [comment]
Resolved!

I found the old tabs settings in the config:

```xml
<rule ref="Squiz.Functions.MultiLineFunctionDeclaration">
		<properties>
			<property name="indent" value="1"/>
		</properties>
	</rule>
```

**Comment by aik099** [comment]
You use spaces for indentation and want use tabs just for multi-line function declaration indentation or other way around?

**Comment by TomasVotruba** [comment]
I moved from tabs to spaces recently.
I just forgot there was manual settings for tabs in the config I had to remove.

**Comment by aik099** [comment]
Then you did wrong, because instead of setting `indent` setting for a sniff to 1 you should have set `--tab-width` CLI option to 4. Then all would be fixed automatically.

Since PHPCS 2.x you can do that in `ruleset.xml` of your standard as well by adding `<arg name="tab-width" value="4" />` in there.

**Comment by TomasVotruba** [comment]
I know, the issue is resolved.

**Comment by aik099** [comment]
Probably that `indent` setting was added there by mistake instead of specifying `tab-width` option back in the days. Now, when you're using spaces the `tab-width` CLI (or ruleset) option is a must. Especially if you're using auto-fixing.
